src/main/kotlin/entry/dsm/equusrecruitment/domain/application/service/InternApplicationService.kt

```diff
@@ -0,0 +1,34 @@
+package entry.dsm.equusrecruitment.domain.application.service
+
+import entry.dsm.equusrecruitment.domain.application.domain.Application
+import entry.dsm.equusrecruitment.domain.application.exception.ApplicationAlreadyExistException
+import entry.dsm.equusrecruitment.domain.application.presentation.dto.request.InternApplicationRequest
+import entry.dsm.equusrecruitment.domain.application.repository.ApplicationRepository
+import jakarta.transaction.Transactional
+import org.springframework.stereotype.Service
+
+@Service
+class InternApplicationService(
+    private val applicationRepository: ApplicationRepository,
+) {
+    @Transactional
+    fun applicationRegister(request: InternApplicationRequest) {
+        val application =
```

Comment:
```suggestion
        val application = 
        applicationRepository.findBySchoolNumber(request.schoolNumber) ?:
        throw ApplicationAlreadyExistException
```
let으로 이 부분까지 null check하는건 옳지 않다고 생각해요 
https://tourspace.tistory.com/208
이 자료가 도움이 될거에요

Reply:
[f3f2aa8](https://github.com/EntryDSM/Equus-Recruitment/pull/4/commits/f3f2aa80e27f2741a2f82ba513166d48fe0e36e0)

제가 코드를 잘못 작성했던것 같습니다. 제가 의도한 바는 이미 동일한 학번의 학생이 신청을 제출한경우 신청이 존재한다면 존재한다는 예외를 반환하는것이 제 목적이였습니다. 이를 수정하였습니다.